### PR TITLE
141 fix wiki screen design

### PR DESCRIPTION
**Pull request by haakonjacobsen**

### CODE REVIEW

**Comment by haakonjacobsen** [comment]
Resolve #141

File: frontend/screens/WikiScreen.tsx [hunk 1]
```diff
@@ -81,10 +80,7 @@ function WikiScreen({ route }: DogPostScreenProps) {
       style={defaultStyles.fullScreen}
       refreshControl={<RefreshControl refreshing={isFetchingBreedDetails} onRefresh={onRefresh} />}
     >
-      <ImageBackground
-        source={{ uri: breedToPicture[dogBreed.name] }}
-        style={styles.breedPicture}
-      />
+      <ImageBackground source={{ uri: wikiData.images[0] }} style={styles.breedPicture} />
```
Comment (AndreasWintherMoen) [inline frontend/screens/WikiScreen.tsx:83]: Can wikiData.images be empty?

**Review by AndreasWintherMoen — APPROVED** [review]

File: frontend/screens/WikiScreen.tsx (hunk 1 above)
Comment (haakonjacobsen) [reply frontend/screens/WikiScreen.tsx:83]: It should never be empty, but I can add a fallback

Comment (haakonjacobsen) [reply frontend/screens/WikiScreen.tsx:83]: This will change when the image slider component is added #145

**Pull request merged**